app/views/users/index.html.erb

```diff
@@ -19,8 +19,8 @@
                     <p class="card-subtitle ">
                       <%= link_to user.name, user_path(user) %>
                     </p>
-                    <%= form_for(current_user.active_friendships.build) do |f| %>
-                      <%= hidden_field_tag :passive_friend_id, user.id %> 
+                    <%= form_for(current_user.outgoing_pending_requests.build) do |f| %>
```

Comment:
**[REQUIRED]**
* Let's separate code logic from the views([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png) from Pathwright).